packages/sources/src/styling/components/link-list.scss

```diff
@@ -0,0 +1,13 @@
+@import "../variables";
```

Comment:
In `@dso-toolkit/sources` maken we alleen mixins. Die mixins worden vanuit een implementatie aangeroepen. In dit geval zal dat alleen `@dso-toolkit/css` zijn, er zal (voorlopig?) geen Web Component voor worden gemaakt

Reply:
ok